ceph/iscsi/initiator.py

```diff
@@ -0,0 +1,122 @@
+from ceph.ceph_admin.common import config_dict_to_string
+
+
+class ISCSIConfigs:
+    """iSCSI Configuration attributes."""
+
+    INITIATOR_IQN = "InitiatorName={IQN}"
```

Comment:
IMHO, we should treat this as an instance property due to its dynamic nature.

Reply:
with IQN, I will try to make it. Or maybe I could take this next improvement PR.

Reply:
Well, I was Just Going through the code, this is the content which would be added to initiator config file, Not really the attribute. for IQN, we have `self.iqn` already.